package.json

```diff
@@ -60,6 +60,9 @@
     "@babel/core": "^7.19.6",
     "@commitlint/cli": "^17.3.0",
     "@commitlint/config-conventional": "^17.3.0",
+    "@fortawesome/fontawesome-svg-core": "^6.2.1",
```

Comment:
I've restored those dependencies because the pipeline was failing. We talked a few weeks ago that they shouldn't be dependencies but we'll do some research later in other PR.

Reply:
Ok. I can help with that research